src/main/java/com/studycollaboproject/scope/controller/PostRestController.java

```diff
@@ -71,7 +70,10 @@ public ResponseDto editPost(
             @Parameter(hidden = true) @AuthenticationPrincipal UserDetails userDetails
     ) {
         log.info("POST, [{}], /api/post/{}, requestDto={}", MDC.get("UUID"), postId, postRequestDto.toString());
-        return postService.editPost(postId, postRequestDto);
+        if (userDetails == null) {
+            throw new RestApiException(ErrorCode.NO_AUTHENTICATION_ERROR);
+        }
+        return postService.editPost(postId, postRequestDto,userDetails.getUsername());
```

Comment:
Service에서 바로 ResponseDto 를 반환하게 하는 것 보다는 Data에 들어갈 객체를 반환해서 넣는게 테스트할 때 더 좋지 않을까요?

예를 들면 이렇게요..!
```suggestion
        PostResponseDto responseDto = postService.editPost(postId, postRequestDto,userDetails.getUsername());
        
        return new ResponseDto("200", "", responseDto);
```
 data 내용에 안들어가더라도 객체를 반환해주는게 테스트할 때 더 편했습니다